Tests/Containers/LoginScreenTest.js

```diff
@@ -41,27 +41,24 @@ test('component structure', t => {
 })
 
 test('sets errors from the props', t => {
-  const wrapper = shallow(<LoginScreen error='FOOBAR ERROR TEXT' />)
+  wrapper.setProps({error: 'FOOBAR ERROR TEXT'})
   t.is(wrapper.containsMatchingElement(<Text>FOOBAR ERROR TEXT</Text>), true)
 })
 
 test('attempts to login', t => {
-  const attemptLoginFn = sinon.stub()
-  const wrapper = shallow(<LoginScreen attemptLogin={attemptLoginFn} error='FOOBAR ERROR TEXT' />)
+  const attemptLogin = sinon.stub()
+  wrapper.setProps({attemptLogin})
   const form = wrapper.childAt(2)
   const buttons = form.childAt(1)
   const firstButton = buttons.childAt(0)
   firstButton.simulate('press')
-  t.true(attemptLoginFn.called)
+  t.true(attemptLogin.called)
 })
 
-// TODO: Figure out how to work with routing.
-// test('direct the user to the verify page of successful login', t => {
-//   const ActionsSpy = sinon.stub(Actions, 'verify')
-//   const wrapper = shallow(<LoginScreen />)
-//   wrapper.setState({ isAttemptingLogin: true})
-//   wrapper.setProps({ fetching: false, error: false})
-//   console.log(ActionsSpy)
-//   t.true(ActionsSpy.called)
-
-// })
+test('direct the user to the verify page after successful login', t => {
+  const VerifyStub = sinon.stub(NavigationActions, 'verify')
```

Comment:
I was stubbing out `Actions` before but it was aliased as `NavigationActions` which is strange to me.  `verify` could not be stubbed on `Actions` even though it is an alias.